ansible/roles/debops.ntp/tasks/install.yml

```diff
@@ -2,14 +2,12 @@
 
 - name: Install required packages
   apt:
-    name: '{{ item }}'
+    name: '{{ "chrony" if (ntp__daemon == "chrony") else ""
+            + "ntpdate" if (ntp__daemon == "ntpdate") else ""
+            + "ntp" if (ntp__daemon == "ntpd") else ""
+            + "openntpd" if (ntp__daemon == "openntpd") else "" }}'
```

Comment:
You should probably use YAML lists for these:
```yaml
    name: '{{ ["chrony"] if (ntp__daemon == "chrony") else []
￼	            + ["ntpdate"] if (ntp__daemon == "ntpdate") else []
￼	            + ["ntp"] if (ntp__daemon == "ntpd") else []
￼	            + ["openntpd"] if (ntp__daemon == "openntpd") else [] }}'
```

Reply:
I used an empty string cause an empty dict won't work for me.
With the requested changes, `debops service/ntp -l myhost` gave me this : 
```
TASK [debops.ntp : Install required packages] ******************************************************************************************
fatal: [vm203030.univ]: FAILED! => {"msg": "Unexpected templating type error occurred on ({{ \"chrony\" if (ntp__daemon == \"chrony\") else [] + \"ntpdate\" if (ntp__daemon == \"ntpdate\") else [] + \"ntp\" if (ntp__daemon == \"ntpd\") else [] + \"openntpd\" if (ntp__daemon == \"openntpd\") else [] }}): can only concatenate list (not \"str\") to list"}
```